src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -199,8 +199,7 @@ internal void OnCodeElementCreated(SyntaxNodeKey nodeKey, EnvDTE.CodeElement ele
             // If we're creating an element with the same node key as an element that's already in the table, just remove
             // the old element. The old element will continue to function but the new element will replace it in the cache.
 
-            EnvDTE.CodeElement oldElement;
-            if (_codeElementTable.TryGetValue(nodeKey, out oldElement))
+            if (_codeElementTable.ContainsKey(nodeKey))
```

Comment:
Would it have made more sense just to have a "Replace" method? Given there's sufficient magic going on with it, I still get there's a try-and-then-do pattern.